### DISCUSSION TITLE
Show actual live data

### DISCUSSION THREAD

**Discussion by walteij** [theOehrly/Fast-F1#510] [Q&A] [answered]
I know we have the ability to use signalr as live client, but has anybody actually created some sort of live data dashboard?
To show drivers speed, gear, position etc?

**Comment by Casper-Guo** [comment]
I use these two during races

[racebot](https://racing.recursiveprojects.cloud/)

[dashboard](https://f1.tdjs.tech/)

**Reply by walteij** [reply]
Ah yes, racebot I cloned and am running a copy for [RaceTalk](https://data.racetalk.nl) ;)
f1 tdjs is new to me, will check when there's a session going on.

I of course know about multiviewer as well, and that does some nice things (and shows a lot), but I was wondering if we could do this kind of stuff with Fast-F1 as well, so read the jsonstream and show the speed/throttle/etc of (a selection of) drivers on a website on a discord server somewhere, instead of creating all sorts of nice plots after the race.

And I am not talking about livestreaming any videofeed, just the telemetry. I found [F1-live-data](https://github.com/f1stuff/f1-live-data) that shows a Influxdb/grafana dashboard which is nice, but I was unable to het some historical view, so I'll have to wait for live sessions to see how that will end up.

**Comment by theOehrly** [comment, marked as answer]
The API parser code and how everything works together in FastF1 really isn't suited for this kind of stuff. The focus in FastF1 is on giving accurate data and combining data from multiple sources. This includes automatic error detection, sometimes correction and filling in missing data, which often cannot be done live because it needs future data as well. And I'd rather improve these features further than rewrite everything so it can do two things but none of them well.
FastF1 simply is intended for data analysis, not live visualization.
